android/app/src/main/java/com/mattermost/rnbeta/StartTimeModule.java

```diff
@@ -0,0 +1,183 @@
+package com.mattermost.rnbeta;
+
+import android.app.Application;
+import android.support.annotation.Nullable;
+import android.util.Log;
+
+import com.facebook.react.bridge.Arguments;
+import com.facebook.react.bridge.Promise;
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.bridge.ReactContextBaseJavaModule;
+import com.facebook.react.bridge.ReactMethod;
+import com.facebook.react.bridge.WritableMap;
+import com.facebook.react.bridge.WritableNativeMap;
+import com.mattermost.rnbeta.react_native_interface.AsyncStorage;
+import com.mattermost.rnbeta.react_native_interface.KeysReadableArray;
+import com.mattermost.rnbeta.react_native_interface.ResolvePromise;
+import com.oblador.keychain.KeychainModule;
+import com.reactnativenavigation.NavigationApplication;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+
+public class StartTimeModule extends ReactContextBaseJavaModule {
+
+    static final String TOOLBAR_BACKGROUND = "TOOLBAR_BACKGROUND";
+    static final String TOOLBAR_TEXT_COLOR = "TOOLBAR_TEXT_COLOR";
+    static final String APP_BACKGROUND = "APP_BACKGROUND";
+    static final String DEVICE_SECURE_CACHE_KEY = "DEVICE_SECURE_CACHE_KEY";
+
+    private final Application mApplication;
+
+    public StartTimeModule(Application application, ReactApplicationContext reactContext) {
+        super(reactContext);
+        mApplication = application;
+    }
+
+    @Override
+    public String getName() {
+        return "StartTime";
+    }
+
+    @Nullable
+    @Override
+    public Map<String, Object> getConstants() {
```

Comment:
I was debating on adding this optimization.  Until I realized that it speeds the push notification reply process.

The idea is to export all values we normally call through the native bridge in JS.

Reply:
I'm thinking that this should not be included in the PR as this is only for telemetry right?